discovery-service/src/main/resources/application.yml

```diff
@@ -163,7 +163,7 @@ logging:
         org.springframework: INFO
         org.apache: INFO
         org.apache.http: DEBUG
-        com.netflix: INFO
```

Comment:
Does this print out the whole activity of the eureka? Asking if it will remain in DEBUG mode